sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/tests/ModelSettingsTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Threading.Tasks;
+using Azure.Iot.TimeSeriesInsights.Models;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Azure.Iot.TimeSeriesInsights.Tests
+{
+    public class ModelSettingsTests : E2eTestBase
+    {
+        public ModelSettingsTests(bool isAsync)
+            : base(isAsync)
+        {
+        }
+
+        [Test]
+        public async Task TimeSeriesInsightsClient_ModelSettingsTest()
+        {
+            TimeSeriesInsightsClient client = GetClient();
+
+            // GET model settings
+            Response<TimeSeriesModelSettings> currentSettings = await client.GetModelSettingsAsync().ConfigureAwait(false);
+            currentSettings.GetRawResponse().Status.Should().Be((int)HttpStatusCode.OK);
+            string testName = "testModel";
+
+            // UPDATE model settings
+            Response<TimeSeriesModelSettings> updatedSettingsName = await client.UpdateModelSettingsNameAsync(testName);
+            updatedSettingsName.GetRawResponse().Status.Should().Be((int)HttpStatusCode.OK);
+            updatedSettingsName.Value.Name.Should().Be(testName);
+            // To do: Add a test for updating default Type Id. Need existing Model type to update with associated type Id.
```

Comment:
This comment is here to remind when we add APIs for Type, we can add a verification here for updating with existing Type Id.

Reply:
Can you please add a Task so we won't lose track of it?

Reply:
Thinking out loud not just for this PR but for all of us - For TODO's in new features, it might be good to add a task and put the task Id in the TODO so that it is easy to search and implement when we get to it. What do you think?

Reply:
added a task and id in comment